script/staging/FullStagingDeploy.s.sol

```diff
@@ -459,50 +440,42 @@ contract StagingDeploymentBase is Script {
         vm.stopBroadcast();
     }
 
-    function _transferOwnershipOfMocks() internal {
-        vm.selectFork(mainnet.forkId);
-        vm.startBroadcast();
-
-        MockDaiUsds(daiUsds).transferOwnership(mainnetAlmProxy);
-        MockPSM(psm).transferOwnership(mainnetAlmProxy);
-
-        vm.stopBroadcast();
+    function _runFullDeployment() internal {
     }
 
-    function _runFullDeployment(bool useLiveContracts) internal {
-        // Step 1: Load general configuration
+    function run() public {
+        vm.setEnv("FOUNDRY_ROOT_CHAINID",             "1");
+        vm.setEnv("FOUNDRY_EXPORTS_OVERWRITE_LATEST", "true");
 
-        string memory common = ScriptTools.loadConfig("common");
+        deployer = msg.sender;
 
-        ilk = common.readString(".ilk").stringToBytes32();
+        mainnet = Domain({
+            name   : "mainnet-staging",
+            config : ScriptTools.loadConfig("mainnet-staging"),
+            forkId : vm.createFork(getChain("mainnet").rpcUrl),
+            admin  : deployer
+        });
+        base = Domain({
+            name   : "base-staging",
+            config : ScriptTools.loadConfig("base-staging"),
+            forkId : vm.createFork(getChain("base").rpcUrl),
+            admin  : deployer
+        });
 
         // Ballpark sizing of rate limits, tokens in PSMs, etc
         // Ballpark sizing of USDS to put in the join contracts, PSMs, etc
-        USDC_UNIT_SIZE = common.readUint(".usdcUnitSize") * 1e6;
-        USDS_UNIT_SIZE = common.readUint(".usdsUnitSize") * 1e18;
-
-        // Step 2: Load domain-specific configurations
-
-        CCTP_TOKEN_MESSENGER_MAINNET = mainnet.config.readAddress(".cctpTokenMessenger");
-        CCTP_TOKEN_MESSENGER_BASE    = base.config.readAddress(".cctpTokenMessenger");
+        USDC_UNIT_SIZE = mainnet.config.readUint(".usdcUnitSize") * 1e6;
+        USDS_UNIT_SIZE = mainnet.config.readUint(".usdsUnitSize") * 1e18;
 
-        SAFE_MAINNET = mainnet.config.readAddress(".safe");
-        USDC         = mainnet.config.readAddress(".usdc");
+        // Run deployment scripts after setting storage variables
 
-        SAFE_BASE = base.config.readAddress(".safe");
-        USDC_BASE = base.config.readAddress(".usdc");
-
-        // Step 3: Run deployment scripts after setting storage variables
-
-        _setUpDependencies(useLiveContracts);
+        _setUpDependencies();
         _setUpAllocationSystem();
         _setUpALMController();
         _setUpBasePSM();
```

Comment:
This can be removed, can use live PSM

Reply:
Ah ya good call. Removed a bunch of other mocks in favour of real one.